manifest.yaml

```diff
@@ -123,6 +123,12 @@ arguments:
       items:
         type: string
     description: Dependencies your repository needs, but not always (e.g. E2E tests).
+  pagerduty.p1:
```

Comment:
I thought that we want to scrape this data from https://github.com/getoutreach/cloud-governance-terraform/blob/main/teams.json

Reply:
+1, I think we want to load this data automatically and stop having it be set